### ISSUE TITLE
question: I have customised $HOME layout, how do I set the Directory plugin to reflect this?

### ISSUE THREAD

**Issue by alan12345678** [lxqt/lxqt#1784]
Finally got to a fully functioning LXQT setup. Have been busy customising.

I'm using "Directory Menu" launchers a great deal. I note that by removing certain default dirs from $HOME the launcher picks up on this. But I want to have quick access from the panel to locations such as $XDG_CONFIG_HOME which is correctly set by `xdg-user-dirs-update.service` to `$HOME/.config` but this does not appear to be picked up in the launcher on login.

How do I add "hidden" files, or indeed any further locations I wish?

##### Steps to Reproduce (for bugs)

1. 
2. 
3. 
4. 

##### System Information

* Distribution & Version: 
* Kernel: 
* Qt Version: 
* liblxqt Version: 
* Package version:

**Comment by tsujan** [comment]
By asking personal questions in LXQt bug tracker, you make devs' jobs harder. As you may have already noted, your reports don't fit into the template either.

Please wait until LXQt forum is fixed and ask your questions there. It's down now but I'm sure that's temporary.

Or you could do it in an Arch forum.

**Comment by alan12345678** [comment]
sure okay - totally appreciate that.

You have illuminated a part of this entire business - namely that ALL of my questions (and which you have been kind enough to respond to) should have been directed to the forum. But umm... the one guy holds the keys to the kingdom? And he claims he's "been busy"?? and this is for a rapidly expanding software base? And yet somehow, despite he didn't kill himself or die of coronavirus, we're STILL waiting for the site to be restored???

Again, NO offence meant at all. You're just a dev, I get it. I won't disturb you further, but a forum isn't merely a luxury, it's ESSENTIAL. Surely if he can't handle the load, someone else needs the keys.

None of which I suppose has got anything to do with what looks like a brewing row over the QT organisation and their new insistence on licensing (= BIG FAT MONEY!) and so on.

I'll use Reddit for now, and leave you in peace.

p.s. I got my system working by restoring the machine (again) to bare Arch and then simply using `pacman` to install the 0.14 release version. Know what? It worked PERFECTLY the instant I logged in. So, I've just developed an allergy to git packages for the most part, but do look forward to the next official release.

:)

**Comment by tsujan** [comment]
No problem is solved by ranting — which was so long-winded that I didn't read.

**Comment by alan12345678** [comment]
Many years ago, in what I shall call "a different life" I ran an estate agency in southern Spain.

From what are called humble beginnings it ended up being (as much by luck as judgement) quite successful. With that success came a huge increase in traffic mostly in the form of emails and phone calls.

As I found myself to be in what was now a huge, swirling mass of human endeavour, I taught myself to touch type -- so much the better that I could answer the 100+ emails we received per day. Yes I know, that's a small number to you but it represented a significant turnover for us.

My point? I would sit down at the email every morning, and dutifully answer EVERY email that came my way. And because the business found itself extremely busy, I wrote in what I considered to be a helpful if brief style - the simple imperative of efficacy and the fact that I had plenty other things to get done that day.

As the months passed and numbers more of those same clients passed through my office, they were split into two broad groups: those who appreciated my honesty and directness (I had no designed approach, I simply answered quickly because I felt I had to), and those who would sometimes swear at me for "being so incredibly rude". I was shocked. How, I wondered, could they POSSIBLY have considered me rude?? I'm really busy. I'm just doing my job.

Eventually a project like yours will succeed or fail not because the very last piece of code is finalised. It will succeed or fail according to whether its users can finally "be bothered" to take on the pain and inconvenience that using said product entails. Like you I have a pretty short attention span when it seems I'm being criticised, but I wrote this with nothing but the best intentions.

I think LXQT is a superb invention, I truly do! But if the guy running the forum - an integral, essential part of promoting and supporting ANY product - can't be arsed to actually restore it online, and meanwhile the long-suffering people like yourselves end up having to bat away people like me... people who are equallly kind, thoughtful and intelligent, but just want to know "how do I fix this?" well... it will end up killing the project dead not because it's a bad product, but because it's just easier to do something else instead.
